### DISCUSSION TITLE
can anyone tell what is going on here? I am pointing it to the right known host file yet i cannot seem to ssh. ssh verbose below

### DISCUSSION THREAD

**Discussion by SwitchSensei** [Eugeny/tabby#8816] [Q&A]
"myusername@FVFFX14HQ6LT ~ % ssh -v -i  /Users/myusername/.ssh/known_hosts myusername@jmp01xx
OpenSSH_9.0p1, LibreSSL 3.3.6
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 21: include /etc/ssh/ssh_config.d/* matched no files
debug1: /etc/ssh/ssh_config line 54: Applying options for *
debug1: Authenticator provider $SSH_SK_PROVIDER did not resolve; disabling
debug1: Connecting to jmp01xx port 22.
debug1: Connection established.
debug1: identity file /Users/myusername/.ssh/known_hosts type -1
debug1: identity file /Users/myusername/.ssh/known_hosts-cert type -1
debug1: Local version string SSH-2.0-OpenSSH_9.0
debug1: Remote protocol version 2.0, remote software version OpenSSH_7.4
debug1: compat_banner: match: OpenSSH_7.4 pat OpenSSH_7.4* compat 0x04000006
debug1: Authenticating to jmp01xx:22 as 'myusername'
debug1: load_hostkeys: fopen /Users/myusername/.ssh/known_hosts2: No such file or directory
debug1: load_hostkeys: fopen /etc/ssh/ssh_known_hosts: No such file or directory
debug1: load_hostkeys: fopen /etc/ssh/ssh_known_hosts2: No such file or directory
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: algorithm: curve25519-sha256
debug1: kex: host key algorithm: ssh-ed25519
debug1: kex: server->client cipher: aes128-ctr MAC: umac-64-etm@openssh.com compression: none
debug1: kex: client->server cipher: aes128-ctr MAC: umac-64-etm@openssh.com compression: none
debug1: expecting SSH2_MSG_KEX_ECDH_REPLY
debug1: SSH2_MSG_KEX_ECDH_REPLY received
debug1: Server host key: ssh-ed25519 SHA256:kkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkk(fakekey)
debug1: load_hostkeys: fopen /Users/myusername/.ssh/known_hosts2: No such file or directory
debug1: load_hostkeys: fopen /etc/ssh/ssh_known_hosts: No such file or directory
debug1: load_hostkeys: fopen /etc/ssh/ssh_known_hosts2: No such file or directory
debug1: Host 'jmp01xx' is known and matches the ED25519 host key.
debug1: Found key in /Users/myusername/.ssh/known_hosts:2
debug1: rekey out after 4294967296 blocks
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: SSH2_MSG_NEWKEYS received
debug1: rekey in after 4294967296 blocks
debug1: get_agent_identities: bound agent to hostkey
debug1: get_agent_identities: ssh_fetch_identitylist: agent contains no identities
debug1: Will attempt key: /Users/myusername/.ssh/known_hosts  explicit
debug1: SSH2_MSG_EXT_INFO received
debug1: kex_input_ext_info: server-sig-algs=<rsa-sha2-256,rsa-sha2-512>
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug1: Authentications that can continue: gssapi-keyex,gssapi-with-mic,password,keyboard-interactive
debug1: Next authentication method: keyboard-interactive
(myusername@jmp01xx) Password: 
debug1: Authentications that can continue: gssapi-keyex,gssapi-with-mic,password,keyboard-interactive
(myusername@jmp01xx) Password: "

**Comment by Dorobuta** [comment]
try either -vv or -vvv for even more debug info
